semgrep/tests/e2e/snapshots/test_rule_parser/test_rule_parser__failure__error_messages/bad8/error-in-color.txt

```diff
@@ -6,6 +6,6 @@
 [94m[22m[24m5 | [0m    languages: [python]
 [94m[22m[24m6 | [0m    severity: WARNING
 
-[31m[22m[24mOne of these properties is missing: 'pattern', 'pattern-either', 'pattern-regex', 'patterns'[0m
+[31m[22m[24mOne of these properties is missing: 'pattern', 'pattern-either', 'pattern-regex', 'patterns', 'project-depends-on'[0m
```

Comment:
Do we want those error messages @mmcqd ?

Reply:
We do, because now `project-depends-on` can be there without any of the `pattern` keys